resources/scripts/components/server/files/FileManagerContainer.tsx

```diff
@@ -41,22 +50,30 @@ export default () => {
         clearFlashes('files');
         setSelectedFiles([]);
         setDirectory(hashToPath(hash));
-    }, [ hash ]);
+    }, [hash]);
 
     useEffect(() => {
         mutate();
-    }, [ directory ]);
+    }, [directory]);
 
     const onSelectAllClick = (e: React.ChangeEvent<HTMLInputElement>) => {
         setSelectedFiles(e.currentTarget.checked ? (files?.map(file => file.name) || []) : []);
     };
 
     if (error) {
         return (
-            <ServerError message={httpErrorToHuman(error)} onRetry={() => mutate()}/>
+            <ServerError message={httpErrorToHuman(error)} onRetry={() => mutate()} />
```

Comment:
Remove the space

Reply:
Fixed in https://github.com/pterodactyl/panel/pull/3926/commits/9862de5fd97dade3f9783d4d07a451504e830143